backend/Testing/Fixtures/SendReceiveFixture.cs

```diff
@@ -0,0 +1,62 @@
+using System.IO.Compression;
+using System.Runtime.CompilerServices;
+using Chorus.VcsDrivers.Mercurial;
+using LexCore.Utils;
+using Shouldly;
+using SIL.Progress;
+using Testing.ApiTests;
+using Testing.Services;
+using static Testing.Services.Constants;
+
+namespace Testing.Fixtures;
+
+public class SendReceiveFixture : IAsyncLifetime
+{
+    private readonly DirectoryInfo _templateRepo = new(Path.Join(BasePath, "_template-repo_"));
+    public ApiTestBase AdminApiTester { get; } = new();
+
+    public async Task InitializeAsync()
+    {
+        DeletePreviousTestFiles();
+        await DownloadTemplateRepo();
+        await AdminApiTester.LoginAs(AdminAuth.Username, AdminAuth.Password);
+    }
+
+    public Task DisposeAsync()
+    {
+        return Task.CompletedTask;
+    }
+
+    private static void DeletePreviousTestFiles()
+    {
+        if (Directory.Exists(BasePath)) Directory.Delete(BasePath, true);
+    }
+
+    private async Task DownloadTemplateRepo()
+    {
+        await using var stream = await AdminApiTester.HttpClient.GetStreamAsync("https://drive.google.com/uc?export=download&id=1w357T1Ti7bDwEof4HPBUZ5gB7WSKA5O2");
+        using var zip = new ZipArchive(stream);
+        zip.ExtractToDirectory(_templateRepo.FullName);
+    }
+
+    public ProjectConfig InitLocalFlexProjectWithRepo([CallerMemberName] string projectName = "")
```

Comment:
I moved this here as previously it was a static method in the Utils class, however it only works if this fixture is used, otherwise the template repo folder will be empty. Moving the function here means that it can't be used incorrectly.